lib/cosmos/microservices/interface_microservice.rb

```diff
@@ -124,7 +124,7 @@ def run
             @interface.write(command)
             CommandTopic.write_packet(command, scope: @scope)
             CommandDecomTopic.write_packet(command, scope: @scope)
-            InterfaceStatusModel.set(@interface.as_json, scope: @scope)
+            # InterfaceStatusModel.set(@interface.as_json, scope: @scope)
```

Comment:
This was updating the command sent counters.   Why comment out?

Reply:
I think this was causing issues for me. I'll re-enable for now and double check later.